filters.go

```diff
@@ -0,0 +1,27 @@
+package aiven
+
+var (
+	// instPassthroughFilter is an instance of passthroughFilter.
+	instPassthroughFilter = new(passthroughFilter)
```

Comment:
We have no use case for this filter ATM; please supply the code snippet we can use in the future. I suggest we remove it for now, and if needs be, we will add it along the way.

Reply:
the previous version of this PR required `PassthroughFilter`, you probably didn't notice it, however it's no longer relevant